S/SCS_GPU/build_tarballs.jl

```diff
@@ -0,0 +1,44 @@
+using BinaryBuilder
+
+name = "SCS"
+version = v"2.1.1"
+
+# Collection of sources required to build SCSBuilder
+sources = [
+    GitSource("https://github.com/cvxgrp/scs.git", "e6ab81db115bb37502de0a9917041a0bc2ded313")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/scs*
+flags="DLONG=0 USE_OPENMP=0 BLAS64=1 BLASSUFFIX=_64_"
+blasldflags="-L${prefix}/lib -lopenblas64_"
+
+CUDA_PATH=$prefix/cuda make BLASLDFLAGS="${blasldflags}" ${flags} out/libscsgpuindir.${dlext}
+
+mkdir -p ${libdir}
+cp out/libscs*.${dlext} ${libdir}
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+
+platforms = [
+    Linux(:x86_64),
+    Windows(:x86_64),
+    MacOS(:x86_64),
+]
```

Comment:
Following a discussion we had with @maleadt a few weeks ago my understanding is that this is going to work only for Linux

Reply:
I see, I've taken my clues from `MAGMA`:
https://github.com/JuliaPackaging/Yggdrasil/blob/54c9c4b4f21ad0968eaf9c9335f9cdb1746d9937/M/MAGMA/build_tarballs.jl#L37

It depends on `CUDA_jll` and builds for all of those platforms... `CUDA_@10.0` builds for those as well:
https://github.com/JuliaPackaging/Yggdrasil/blob/816eae9348367497c65655901ee6ab124ce2dc14/C/CUDA/CUDA%4010.0/build_tarballs.jl#L189

@maleadt could you chip in?

Reply:
but as You have predicted Linux builds fine, MacOSX and Windows fail :laughing:

Reply:
I don't think magna ever worked at all